index.html

```diff
@@ -10,21 +11,137 @@
 </head>
 
 <body>
```

Comment:
The page has a horizontal overflow which means that some content is overflowing out of the container. Please see the below screenshot and fix the issue.
![overflow](https://user-images.githubusercontent.com/35504149/130312181-3b938e6e-75b4-4fa7-be0a-41cecb88b2db.png)

Reply:
I checked and in my browser everything is ok, can you explain more, please.

Reply:
In my browser, it's centered, I tried many times.

Reply:
On my browser it is not, you can check by yourself if you want.